src/identity/identity.controller.spec.ts

```diff
@@ -37,8 +39,9 @@ describe('DidController', () => {
     test('should get a identity document for the url', async () => {
```

Comment:
Test with
 * 0 additional verification methods (implicit DID)
 * 1 additional verification method
 * 2 (or more) additional verification methods

Reply:
done (created `identity.service.spec.ts` for proper unit testing)